### PR TITLE
chore(deps): support thecodingmachine/safe v2

### PR DESCRIPTION
**Pull request by p4veI**
Hi, would it be possible to support thecodingmachine/safe v2 like this in this utility? My projects mostly use v2 already but I wanted to give this thing a try.

### CODE REVIEW

**Comment by staabm** [comment]
I couldnt find breaking changes in v2…?

File: composer.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
         "php": ">=7.4 || ^8.0",
         "nette/neon": "^3.2",
         "symfony/polyfill-php80": "^1.26",
-        "thecodingmachine/safe": "^1.3"
+        "thecodingmachine/safe": "^1.3 || ^2.2"
```
Comment (staabm) [inline composer.json:16]: Any particluar reason to add only 2.2+ and not 2.1 or 2.0 or similar?

**Comment by p4veI** [comment]
> nice to see you again.
was vacationing, will get back to phpstan-dba soon I promise 😄 

> I couldnt find breaking changes in v2…?
Some functions were deprecated, but since you're running the safe rule with phpstan all looks ok with v2. Didn't sneak around much here tho.

File: composer.json (hunk 1 above)
Comment (p4veI) [reply composer.json:16]: not really, 2.0 should be sufficient, I'll change it.

**Comment by staabm** [comment]
> was vacationing, will get back to phpstan-dba soon I promise

cool, hopefully you had a great time. Looking forward to your contributions.

Thank you and have a great day

**Comment by staabm** [comment]
Published as https://github.com/staabm/phpstan-baseline-analysis/releases/tag/0.8

**Pull request merged**